lib/sift.rb

```diff
@@ -275,12 +275,7 @@ def post(endpoint, payload)
 
       #Rails.logger.debug("sift_debug: post: payload = #{payload.inspect}")
 
-      # Account for a '/' or not at start of endpoint
-      if !endpoint.start_with? '/'
-        target = "/#{endpoint}"
-      end
-
-      request_url = "#{@api_url}#{endpoint}"
+      request_url = File.join(@api_url, endpoint)
```

Comment:
How about using `URI.join` instead of `File.join`? The result might be the same, but at least it's more descriptive of what it's doing?

```suggestion
      request_url = URI.join(@api_url, endpoint)
```

Reply:
probably fine for this case as there's only two fragments. URI.join always confused me as `URI.join("http://test.com", "/foo" , "/bar").to_s` results in `http://test.com/bar`

Reply:
Wow, didn't know that! That's really bad, especially since `File.join("http://test.com", "/foo" , "/bar").to_s` does the right thing `http://test.com/foo/bar`